src/sass/navbar.sass

```
@@ -1,6 +1,6 @@
 .navbar
   border: none
-  display: flex
+  display: flexy
```

@TroubleBeast hier sollte display: flex stehen oder ? 
```suggestion
  display: flex
```

---

Fixed